src/InputWithLabel.js

```diff
@@ -0,0 +1,31 @@
+import React, { useRef, useEffect } from "react";
+
+
+const InputWithLabel = (props) => {
+    const inputRef = useRef();
+    useEffect(() => {
+        if (props.todoTitle === "") {
+            inputRef.current.focus();
+            console.log(props.todoTitle);
+        }
+    }, [props.todoTitle]);
+
+    return (
+        <>  <label htmlFor="todoTitle">
+            {props.children}</label>
+            <input type="text"
+                id="todoTitle"
+                name="title"
+                value={props.todoTitle}
+                onChange={props.handleTitleChange}
+                ref={inputRef}
+            >
+
+            </input>
+        </>
```

Comment:
Tags should usually have their own line. I would lay out the above code like this:
```
<>
    <label htmlFor="todoTitle">{props.children}</label>
    <input
        id
        value
        etc.
    />
</>
```
Sometimes this sort of thing depends on the coding style, but here the label should definitely be on a different line to the fragment.